gateway-manager/templates/cors_template.json

```diff
@@ -2,7 +2,7 @@
   "name": "cors",
   "config.credentials": "true",
   "config.exposed_headers": "Authorization",
```

Comment:
shouldn't it be here too? `"config.exposed_headers": "Content-Disposition, Authorization",` :thinking:

Reply:
Ah yeah, I think you're right:
https://docs.konghq.com/hub/kong-inc/cors/#parameters